pwd-service/src/main/resources/static/css/main.css

```diff
@@ -68,17 +69,27 @@ hr {
     padding: 0;
 }
 
+.navbar {
+    min-height: 65px;
+}
+
 .navbar-default {
-    background-color: #3498db;
+    background-color: #ffffff;
     border-color: transparent;
 }
 
 .navbar-default .navbar-brand {
-    color: white;
+    color: black;
```

Comment:
Chyba lepiej designem sie nie zajmuj, tylko napisz co chcesz zrobic to to ogarne:)